demo/common/overview.c

```diff
@@ -584,6 +584,36 @@ overview(struct nk_context *ctx)
                         nk_combo_end(ctx);
                     }
                 }
+                {
+                    /* Combobox from array of struct */
+                    static int sel=0;
+                    struct student {
+                        int id;
+                        char* name;
+                        int age;
+                        char* major;
+                    };
+
+                    static struct student students[4] = {
+                        {0, "----", 0, "----"},
+                        {1, "Mike", 20, "CS"},
+                        {2, "Jim", 19, "Maths"},
+                        {3, "Julia", 20, "Biology"}
+                    };
```

Comment:
I believe for static definitions, those may want to be outside as well. In addition, using this format may make it easier to read...

```c
static struct student students[4] = {
  {
    .id = 0,
    .name = "---"
    .age = 0,
    .major = "---"
  },
  ...
```

In addition, what is the first `----` entry for?